lib/manager/python/extract.js

```diff
@@ -0,0 +1,16 @@
+
+module.exports = {
+  extractDependencies,
+};
+
+function extractDependencies(packageContent) {
+  const regex = /^([a-z][-\w.]+)==([\d.]+)/ig;
+  return packageContent.split('\n')
+    .map(l => regex.exec(l))
+    .filter(Boolean)
+    .map((matches) => ({
+      depName: matches[1],
+      depType: 'python',
+      currentVersion: matches[2]
```

Comment:
Should we filter based on exact versions only to begin? e.g. ignore versions like `2.6` until we've written the code to properly understand python ranges?